app/src/main/java/com/hedvig/app/feature/birthday/viewmodel/BirthdayViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.hedvig.app.feature.birthday.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.hedvig.app.feature.birthday.data.BirthdayRepository
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import timber.log.Timber
+import java.util.*
+
+
+class BirthdayViewModel(
+    private val birthdayRepository: BirthdayRepository
+) : ViewModel() {
+
+    private lateinit var birthDate: String
```

Comment:
This value only seems to be used in the `isBirthday`-function. We could perhaps remove it from the ViewModel and make it a parameter for that function, making it a little bit more pure 🙂

Reply:
`birthDate: String` is used in both `isBirthday()` and `getBirthday()`

Reply:
That's true! However, `getBirthday()` only uses it to write to the property, and `isBirthday()` then subsequently reads from it. `isBirthday()` could be rewritten to have the signature `isBirthday(birthdate: String)`, and as such we avoid the property in the ViewModel 🙂